hooks/usePrimarySalePrice.tsx

```diff
@@ -0,0 +1,51 @@
+import { formatUnits } from 'ethers/lib/utils'
+
+import { first } from 'lodash'
+
+import { LilNounsAuctionEventTypes, NounsAuctionEventTypes } from '@noun-auction'
+
+import { useIsAuctionSettled } from './useIsAuctionSettled'
+
+// FIXME
+function getLatestBlockNumberForEvent(a: any, b: any) {
+  return a.transactionInfo.blockNumber &&
+    b.transactionInfo.blockNumber &&
+    a.transactionInfo.blockNumber > b.transactionInfo.blockNumber
+    ? -1
+    : 1
+}
+
+type Props = {
+  collectionAddress: string
+}
+
+export const usePrimarySalePrice = ({ collectionAddress }: Props) => {
+  // FIXME: use contractAddress to get market type and events
+  const marketType = 'NOUNS_AUCTION'
+  const events = { nodes: [] }
+
+  const isSettled = useIsAuctionSettled({ marketType, events })
+  const noAuctionHistory = events?.nodes.length === 0
+
+  if (noAuctionHistory || !isSettled) return { primarySalePrice: '0' }
+
+  const bidEventType =
+    marketType === 'NOUNS_AUCTION'
+      ? NounsAuctionEventTypes.NOUNS_AUCTION_HOUSE_AUCTION_BID_EVENT
+      : LilNounsAuctionEventTypes.LIL_NOUNS_AUCTION_HOUSE_AUCTION_BID_EVENT
```

Comment:
Have types for this been normalized in the database? Ideally events would be of the same type for similar events